doc/source/security/data-scrubbing.rst

```diff
@@ -5,26 +5,20 @@ Data Scrubbing
 
 :Availability: This feature is **Experimental** quality
 
-While data encryption ensures that the existing data are not stored in plain
-form, the data scrubbing literally removes the data once the user decides they
-should be deleted. Compare this behavior with how the ``DELETE`` statement works
-which only marks the affected data as *deleted* - the space claimed by this data
-is overwritten with new data later.
+When you delete data with ``DELETE``, the operating system only marks the data as *deleted*. The data remains in the system. New data will eventually overwrite the data later. Using immediate scrubbing, you remove the data and reclaim the space immediately. With background scrubbing, a background thread removes the data and reclaims the space possibly later.
```

Comment:
remove the word later after ovewrite the data (eventually gets context that it will happen)

Remove "possibly later" if necessary add context of when.

Reply:
Rewritten.